### ISSUE TITLE
Tests for DiseasystoreGoogleCovid19 fails on postgres backends

### ISSUE THREAD

**Issue by RasmusSkytte** [ssi-dk/diseasystore#94]
**Describe the bug**
For PostgreSQL back ends, the tests fails with the following error:

```
── Error ('test-DiseasystoreGoogleCovid19.R:172:7'): DiseasystoreGoogleCovid19 can extend existing features ──
<purrr_error_indexed/rlang_error/error/condition>
Error in `map2(.x, .y, .f, ..., .progress = .progress)`: i In index: 7.
Caused by error in `dplyr::collect()`:
! Failed to collect lazy table.
Caused by error:
! Inactive result set
```

**To Reproduce**
Run the actions on Github

**Expected behaviour**
No errors should be produced

**Screenshots**
N/A

**Provide output from `sessionInfo()`**
N/A

* [x] I have checked that the bug is not already described in [open issues](https://github.com/ssi-dk/diseasystore/issues?q=is%3Aopen)

**Additional context**
N/A

**Comment by RasmusSkytte** [comment]
The tests seems to fail intermittently. Still investigating.

**Comment by RasmusSkytte** [comment]
Errors have not been seen for a while. Closing this issue

**Comment by RasmusSkytte** [comment]
Tests fail again:

Here is a failed action:

https://github.com/ssi-dk/diseasystore/actions/runs/7423309903/job/20250342436?pr=110

**Comment by RasmusSkytte** [comment]
Development version of [SCDB](https://github.com/ssi-dk/SCDB) has fixes to this particular issue

(Although not listed in its NEWS.md)

**Comment by RasmusSkytte** [comment]
Thus should be fixed now with the latest release of SCDB (v0.3)
